src/Character/styles.module.scss

```diff
@@ -2,13 +2,17 @@
   --background: #000;
   --character-width: '1rem';
   --border-width: '1px';
+  --border-radius: '10px';
+  --margin: '10px';
 
   background: var(--background);
   display: flex;
   justify-content: center;
   width: var(--character-width);
   padding: 0.5em;
   position: relative;
+  border-radius: var(--border-radius);
+  margin-right: var(--margin);
```

Comment:
As it is I think this overlaps in function a bit with `border-width`. I understand why you'd want to break the character spacing away from border-width, so two requests:
1. consider renaming this prop to something more descriptive like `characterSpacing`, css variable name `--character-spacing`
2. Implement this here and replace the border-left with margin-left and this variable: https://github.com/robonyong/react-split-flap-display/blob/master/src/SplitFlapDisplay/styles.module.scss#L13